test/parallel/test-runner-output.mjs

```diff
@@ -80,6 +103,7 @@ const tests = [
   { name: 'test-runner/output/spec_reporter_successful.js', transform: specTransform },
   { name: 'test-runner/output/spec_reporter.js', transform: specTransform },
   { name: 'test-runner/output/spec_reporter_cli.js', transform: specTransform },
+  skipIfNoInspector ? false : { name: 'test-runner/output/lcov_reporter.js', transform: lcovTransform },
```

Comment:
If this is written consistently with the last test in this array (which also skips if the inspector is not available) then `skipIfNoInspector` isn't needed.

```suggestion
  process.features.inspector ? { name: 'test-runner/output/lcov_reporter.js', transform: lcovTransform } : false,
```

Reply:
Good spot, I hadn't noticed that last test skipping for the same reason.